### DISCUSSION TITLE
资源中心上传文件大小限制

### DISCUSSION THREAD

**Discussion by zhangpengzsl** [apache/dolphinscheduler#6796] [Q&A]
dolphinscheduler on k8s部署，往资源中心上传文件的时候，大小超过1M的就上传不上去，dolphinscheduler-api中没有错误日志，通过f12查看控制台出现 uncaught exception: Object。有同学知道怎么解决吗？

**Comment by davidzollo** [comment]
as a global project, please describe it in English, thx

**Reply by zhangpengzsl** [reply]
well. I will create a new discussion and give my solution.

**Reply by gaoyan1998** [reply]
请问你解决了吗，我也遇到了这个问题
